prefect_docker/worker.py

```diff
@@ -523,6 +533,7 @@ def _build_container_settings(
             mem_limit=configuration.mem_limit,
             memswap_limit=configuration.memswap_limit,
             privileged=configuration.privileged,
+            **extra_args
```

Comment:
We will need to sanitize this value to ensure we aren't passing in duplicate kwargs.